p/python-gbinder/stone.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : python-gbinder
+version     : 1.1.2
+release     : 1
+homepage    : https://github.com/erfanoabdi/gbinder-python
+upstreams   :
+    - https://github.com/erfanoabdi/gbinder-python/archive/1.1.2/gbinder-python-1.1.2.tar.gz : 2dc424d5c2594146612e4bd752964f8928a62eec7c5ce6046f4c582079d0b537
+license     : GPL-3.0-only
+summary     : Python bindings for libgbinder
+description : |
+    Python bindings for libgbinder
+builddeps   :
+    - pkgconfig(libgbinder)
+    - cython
+    - python-setuptools
```

Comment:
`binary(cython)`, and setuptools can be removed from the builddeps since it's pulled in automatically by `%python_setup`